frontend/src/CrosswordSelector.css

```diff
@@ -0,0 +1,48 @@
+.crosswordselector {
+  width: 100%;
+  max-width: 1024px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 348px);
+  column-gap: 48px;
+  row-gap: 48px;
+}
+.crosswordselector-card {
+  width: 300px;
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
+  cursor: pointer;
+  margin: 0 48px 0 0;
```

Comment:
Is this needed when we can use `grid-column-gap` on the CSS grid?

Reply:
Removed